Config/CMakeIncludes/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+    if (Tasmanian_ENABLE_FORTRAN)
+        set(Tasmanian_post_install "${Tasmanian_post_install}                  ${CMAKE_SHARED_LIBRARY_PREFIX}tasmanianfortran90${CMAKE_SHARED_LIBRARY_SUFFIX}\n")
+    endif()
+endif()
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+    if (Tasmanian_STATIC_LIBRARY)
+        set(Tasmanian_post_install "${Tasmanian_post_install}static libraries: ${CMAKE_STATIC_LIBRARY_PREFIX}tasmaniansparsegrid_static${CMAKE_STATIC_LIBRARY_SUFFIX}\n")
+        set(Tasmanian_post_install "${Tasmanian_post_install}                  ${CMAKE_STATIC_LIBRARY_PREFIX}tasmaniandream_static${CMAKE_STATIC_LIBRARY_SUFFIX}\n")
+        if (Tasmanian_ENABLE_FORTRAN)
+            set(Tasmanian_post_install "${Tasmanian_post_install}                  ${CMAKE_STATIC_LIBRARY_PREFIX}tasmanianfortran90_static${CMAKE_STATIC_LIBRARY_SUFFIX}\n")
+        endif()
+    endif()
+else()
+    if (Tasmanian_STATIC_LIBRARY)
+        set(Tasmanian_post_install "${Tasmanian_post_install}static libraries: ${CMAKE_STATIC_LIBRARY_PREFIX}tasmaniansparsegrid${CMAKE_STATIC_LIBRARY_SUFFIX}\n")
+        set(Tasmanian_post_install "${Tasmanian_post_install}                  ${CMAKE_STATIC_LIBRARY_PREFIX}tasmaniandream${CMAKE_STATIC_LIBRARY_SUFFIX}\n")
+        if (Tasmanian_ENABLE_FORTRAN)
+            set(Tasmanian_post_install "${Tasmanian_post_install}                  ${CMAKE_STATIC_LIBRARY_PREFIX}tasmanianfortran90${CMAKE_STATIC_LIBRARY_SUFFIX}\n")
+        endif()
+    endif()
+endif()
+set(Tasmanian_post_install "${Tasmanian_post_install}C++ examples:     ${CMAKE_INSTALL_PREFIX}/share/Tasmanian/examples/example_sparse_grids.cpp\n")
+set(Tasmanian_post_install "${Tasmanian_post_install}                  ${CMAKE_INSTALL_PREFIX}/share/Tasmanian/examples/example_dream.cpp\n")
+if (Tasmanian_ENABLE_FORTRAN)
+    set(Tasmanian_post_install "${Tasmanian_post_install}Fortran examples: ${CMAKE_INSTALL_PREFIX}/share/Tasmanian/examples/example_sparse_grids.f90\n")
+endif()
+set(Tasmanian_post_install "${Tasmanian_post_install}                  see CMakeLists.txt in ${CMAKE_INSTALL_PREFIX}/examples\n\n")
+
+set(Tasmanian_post_install "${Tasmanian_post_install}bash export commands (add to your environment, i.e., ~/.bashrc):\n\n")
+set(Tasmanian_post_install "${Tasmanian_post_install}export PATH=$PATH:${CMAKE_INSTALL_PREFIX}/bin/\n")
+set(Tasmanian_post_install "${Tasmanian_post_install}export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${CMAKE_INSTALL_PREFIX}/lib/\n")
+
+set(Tasmanian_post_install "${Tasmanian_post_install}above commands are writte in the following\n")
+set(Tasmanian_post_install "${Tasmanian_post_install}bash source files: ${CMAKE_INSTALL_PREFIX}/config/TasmanianENVsetup.sh\n")
+set(Tasmanian_post_install "${Tasmanian_post_install}cmake import:             ${CMAKE_INSTALL_PREFIX}/lib/Tasmanian/Tasmanian.cmake\n")
+set(Tasmanian_post_install "${Tasmanian_post_install}cmake directive:          include(\\\"${CMAKE_INSTALL_PREFIX}/lib/Tasmanian/Tasmanian.cmake\\\")\n")
```

Comment:
triple escape?  oh boy!

Reply:
Yep, I want `" "` to appear in code, so I had to escape it, but it gets processed twice, one when the variable is set, once when the post-install message is printed. Hence, I need to escape the escape character.